docs/en/logs/logs-overview.asciidoc

```diff
@@ -2,9 +2,9 @@
 [role="xpack"]
 == Logs monitoring overview
 
-Logs monitoring enables you to view logs from your infrastructure and identify problems in near real time.
+Logs monitoring enables you to view logs from your infrastructure to help identify problems in real-time.
 You can view logs from servers, containers, services, and so on.
-Then you can drill down to view more detailed information about an individual log entry, or you can seamlessly switch to view corresponding metrics, uptime information or APM traces where available. You can also use machine learning to automatically detect some kinds of log anomalies.
+Additionally, you can drill down to view detailed information about an individual log entry, or you can seamlessly switch to view corresponding metrics, uptime information, or APM traces where available. You can also use machine learning to detect specific log anomalies automatically.
 
 // Add links to metrics, uptime and APM when I have good places to link to.
 
```

Comment:
Can you also update the `image::images/logs-monitoring-architecture.png[]` to this image:

https://user-images.githubusercontent.com/20965076/67385217-fb338a00-f592-11e9-9bad-95d083dfed1a.png

Reply:
FYI, the updated image is from https://github.com/elastic/observability-dev/issues/378#issuecomment-545384684.

Reply:
Screenshot updated.